src/scythe-bidder/constants.ts

```diff
@@ -45,3 +45,9 @@ export const MAX_PLAYERS = 7;
 
 export const MAX_PLAYERS_BASE = 5;
 export const MAX_PLAYERS_IFA = 7;
+
+const MAT_IMAGES:  { [key: string]: string } = {};
+MATS_IFA.forEach((mat,idx) => {
+  MAT_IMAGES[mat] = process.env.PUBLIC_URL + "/" + mat + ".png";
+})
+export {MAT_IMAGES};
```

Comment:
This file is used by both the server and the client, and since this will only work on the client, maybe we should just move it back to bidarea while that is the only file using it? I don't know why someone would try to import it server-side, but if they did, `process.env.PUBLIC_URL` wouldn't work correctly, I think.

Reply:
Eventually, this has boiled down a simple change in the href and adding the mat images.